packages/liveblocks-core/src/lib/Poller.ts

```diff
@@ -79,7 +91,31 @@ export function makePoller(
     }
   }
 
+  function pollNowIfStale(maxStaleTimeMs: number) {
+    if (context.state !== "running") {
+      return;
+    }
+
+    // If a poll is already in progress, do nothing
+    if (context.pollPromise !== null) {
+      return;
+    }
+
+    const lastPolledAt = context.lastPolledAt ?? 0;
+
+    if (performance.now() - lastPolledAt > maxStaleTimeMs) {
```

Comment:
To me the conditional would be slightly more readable if we expressed it as:

```suggestion
    if (lastPolledAt + maxStaleTimeMs < performance.now()) {
```

Reply:
Actually, now that I'm looking at the implementation, I think there is a tiny edge case bug here. If there is no initial data, you _always_ want to poll, so you cannot use the `?? 0` construct here. I'll push a fix while at it.

Reply:
Bug illustrated in cbc98aa.
Bug fixed in 5bb8a24.